Lib/test/test_memoryio.py

```diff
@@ -11,6 +11,13 @@
 import pickle
 import sys
 
+class IntLike():
```

Comment:
not worried at all, and I would certainly follow your advice.
just wondered whether this should be added to the PEP..

and it seems that the docs think like you - https://docs.python.org/3.7/tutorial/classes.html#class-definition-syntax.